lib/improver/spotdata/common_functions.py

```diff
@@ -35,6 +35,8 @@
 
 import warnings
 import numpy as np
+from datetime import datetime as dt
+import iris
 from iris import Constraint
 import iris
 from iris.time import PartialDateTime
```

Comment:
Because the module is being imported anyway and it makes my code look nicer :smile:
Is that enough of a reason?